### PR TITLE
travel fund: update travel fund for @ryzokuken (C+L Bengaluru 2018)

### PR DESCRIPTION
**Pull request by ryzokuken**
Refs: https://github.com/nodejs/code-and-learn/issues/88

Breakdown

| Purpose | Amount Requested |
|---|---|
| Airfare | $200 |
| Transportation | $100 |

/cc @gireeshpunathil @thefourtheye 

/cc @nodejs/community-committee @nodejs/tsc

### CODE REVIEW

**Comment by ryzokuken** [comment]
Hi @Trott! I usually leave some amount as "miscellaneous" that I can draw from in case of unexpected costs or some of the other costs increasing beyond their usual limits (eg: super high transportation costs, sudden spike in airfare, etc.). I do this so that no matter what, I don't go over the upper limit that I initially request since I'm more comfortable requesting for more and ending up using less.

Personally, I think the final costs shouldn't really go above $200.

Would you be more comfortable if I requested for the lower cap instead?

**Comment by joyeecheung** [comment]
@ryzokuken if it’s all transportation you can just put it as transportation - the foundation staff will validate the receipts and stuff eventually anyway, I believe

**Comment by ryzokuken** [comment]
@joyeecheung Umm, that was way too obvious in hindsight, sorry. Made the change.

**Comment by gireeshpunathil** [comment]
don't know whether my review is of any significance in this repo; still providing my LGTM as the co-ordinator of the Code & Learn event. thanks!

File: TravelFunds/2018.md [hunk 1]
```diff
@@ -46,3 +46,4 @@ Waleed	|	Ashraf	|	JS Interactive and Collaborator Summit	|	Attendance, Collabora
 Saleh	|	Abdel Motaal	|	JS Interactive and Collaborator Summit	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, Canada	|	10Oct -14Oct 2018	|	US$1700	| 19Sep 2018 | https://github.com/nodejs/admin/pull/232 | | | | |	
 Anatoli	|	Papirovski	|	JS Interactive and Collaborator Summit	|	Attendance, Collaborate and Code & Learn	|	Vancouver BC, Canada	|	9Oct - 13Oct 2018	|	US$1633.53	|	26Sep 2018	|	https://github.com/nodejs/admin/pull/235 |		|		|		|		|	
 Dan | Shaw | JS Interactive and Collaborator Summit | Attendance, Collaborate and Code & Learn | Vancouver, BC, CAN | 9oct - 14oct 2018 | US$1820 | | https://github.com/nodejs/admin/pull/224 | | | | |
+Ujjwal | Sharma | Code & Learn, Bengaluru | Collaborate and Code & Learn | Bengaluru, India | 17th Nov 2018 | US$300 | 5th Sep 2018 | | | | | |
```
Comment (thefourtheye) [inline TravelFunds/2018.md]: Nit: The pull request date should be updated to 5th Nov 2018 I guess.

Comment (ryzokuken) [reply TravelFunds/2018.md]: @thefourtheye I thought this was optional. I could update this before landing?

**Comment by ryzokuken** [comment]
@nodejs/community-committee could someone please look into this? The event is less than a week away and I'd need to start making arrangements.

**Comment by ryzokuken** [comment]
Now that this has the required amount of approvals, could someone please merge this in so that I could finish preparations? Thanks.

**Pull request merged**